app/graphql/types/champs/repetition_champ_type.rb

```diff
@@ -11,5 +16,15 @@ def champs
         Loaders::Association.for(object.class, :champs).load(object)
       end
     end
+
+    def rows
+      if object.champs.loaded?
+        object.champs.group_by(&:row).values.map { { champs: _1 } }
```

Comment:
```suggestion
        object.rows.map { { champs: _1 } }
```
cf: https://github.com/demarches-simplifiees/demarches-simplifiees.fr/blob/main/app/models/champs/repetition_champ.rb#L26

Reply:
j'imagine que les champs du rows garde le bon ordre [en gros a chaque row, les champs sont dans le mm ordre]